### PR TITLE
Update ingress-controllers.md with OCI Native Ingress Controller

### PR DESCRIPTION
**Pull request by Inbaraj-S**
Adding the reference to the OCI Native Ingress Controller to the public documentation.
https://kubernetes.io/docs/concepts/services-networking/ingress-controllers/

### CODE REVIEW

**Review by mengjiao-liu — CHANGES_REQUESTED** [review]

File: content/en/docs/concepts/services-networking/ingress-controllers.md
[the remark this answers is not in the document]
Comment (Inbaraj-S) [reply content/en/docs/concepts/services-networking/ingress-controllers.md:1]: @sftim 
I will update the supported version part and other review comments in this pr.

**Comment by mengjiao-liu** [comment]
Have you been able to make any changes following the suggested comments? I don't see a difference.

By the way, the commit message should not contain "Merge". Please squash your commits to one.
> In particular, merge commits must be removed, as they will not pass tests.

see [squash commits](https://www.kubernetes.dev/docs/guide/github-workflow#squash-commits).

**Comment by Inbaraj-S** [comment]
Fixed the review comments.
Please check.
@sftim 
@mengjiao-liu

File: content/en/docs/concepts/services-networking/ingress-controllers.md [hunk 1]
```diff
@@ -56,6 +56,7 @@ Kubernetes as a project supports and maintains [AWS](https://github.com/kubernet
 * The [NGINX Ingress Controller for Kubernetes](https://www.nginx.com/products/nginx-ingress-controller/)
   works with the [NGINX](https://www.nginx.com/resources/glossary/nginx/) webserver (as a proxy).
 * The [ngrok Kubernetes Ingress Controller](https://github.com/ngrok/kubernetes-ingress-controller) is an open source controller for adding secure public access to your K8s services using the [ngrok platform](https://ngrok.com).
+* The [OCI Native Ingress Controller](https://github.com/oracle/oci-native-ingress-controller) is an Ingress controller for Oracle Cloud Infrastructure which allows you to manage the [OCI Load Balancer](https://docs.oracle.com/en-us/iaas/Content/Balance/home.htm).
```
Comment (dipesh-rawat) [inline content/en/docs/concepts/services-networking/ingress-controllers.md:59]: Based on the generated [deploy-preview-4153-services-networking/ingress-controllers](https://deploy-preview-41531--kubernetes-io-main-staging.netlify.app/docs/concepts/services-networking/ingress-controllers/#:~:text=ngrok%20platform.-,The,-OCI%20Native%20Ingress), changes LGTM.
 
 _Minor nit (Refer suggestion below)_
```suggestion
* The [OCI Native Ingress Controller](https://github.com/oracle/oci-native-ingress-controller#readme) is an Ingress controller for Oracle Cloud Infrastructure which allows you to manage the [OCI Load Balancer](https://docs.oracle.com/en-us/iaas/Content/Balance/home.htm).
```

**Comment by dipesh-rawat** [comment]
Removing unnecessary labels.
/remove-language de
/remove-language es
/remove-language fr
/remove-language hi
/remove-language id
/remove-language ja
/remove-language ko
/remove-language pt
/remove-language ru
/remove-language vi
/remove-language zh

File: content/en/docs/concepts/services-networking/ingress-controllers.md (hunk 1 above)
Comment (Inbaraj-S) [reply content/en/docs/concepts/services-networking/ingress-controllers.md:59]: Done.

File: content/en/docs/concepts/services-networking/ingress-controllers.md [hunk 2]
```diff
@@ -56,6 +56,7 @@ Kubernetes as a project supports and maintains [AWS](https://github.com/kubernet
 * The [NGINX Ingress Controller for Kubernetes](https://www.nginx.com/products/nginx-ingress-controller/)
   works with the [NGINX](https://www.nginx.com/resources/glossary/nginx/) webserver (as a proxy).
 * The [ngrok Kubernetes Ingress Controller](https://github.com/ngrok/kubernetes-ingress-controller) is an open source controller for adding secure public access to your K8s services using the [ngrok platform](https://ngrok.com).
+* [OCI Native Ingress Controller](https://github.com/oracle/oci-native-ingress-controller) is an Ingress controller for Oracle Cloud Infrastructure which provides the following [features](https://github.com/oracle/oci-native-ingress-controller/blob/main/GettingStarted.md#feature-support).
```
[the remark this answers is not in the document]
Comment (Inbaraj-S) [reply content/en/docs/concepts/services-networking/ingress-controllers.md]: Done

**Comment by Inbaraj-S** [comment]
Fixed the failing build with retrigger, 
Please check.
@sftim
@mengjiao-liu
@dipesh-rawat

**Review by dipesh-rawat — COMMENTED** [review]
Thanks for addressing all the review comments.
Preview: [deploy-preview-4153-services-networking/ingress-controllers](https://deploy-preview-41531--kubernetes-io-main-staging.netlify.app/docs/concepts/services-networking/ingress-controllers/#:~:text=ngrok%20platform.-,The,-OCI%20Native%20Ingress)

/lgtm

**Comment by sftim** [comment]
Based on https://github.com/kubernetes/website/pull/41531#issuecomment-1637519318
/assign @kbhawkey

**Comment by Inbaraj-S** [comment]
@kbhawkey 
Could you please check this minor item and approve.

**Pull request merged**